### DISCUSSION TITLE
essay/2024/05/01/old-photos-museum

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#208] [Comments]
# essay/2024/05/01/old-photos-museum

80년대에 태어나서 아쉬운 점이 있다면 어릴 때 사진이 없다는 것. 카메라가 귀하기도 했습니다만 그나마 가끔 일회용 카메라로 찍었던 사진들도 대부분 분실했습니다. 살면서 계속 후회하게 되는 일 중 하나입니다. 잘 좀 보관할걸. 이렇게 후회할 줄 그때는 몰랐습니다.

https://jeho.page/essay/2024/05/01/old-photos-museum.html

**Comment by natz92** [comment]
사진에 담긴 자신의 모습도 좋고, 그때는 그 장소가 이런 모습이었구나 돌아보는 추억거리가 되는 점이 좋게 느껴집니다.
아직 30 초중반이지만, 태어날 때부터 고등학생 때 까지의 사진첩을 전부 잃어 버렸죠. 그때를 추억할 수 있는 방법은 이제 점점 잊혀져가는 기억 뿐이라 20대 중반 부터는 별거 아닌 사진이라도 매일매일 찍으며 기록하고 있습니다. 너무 많아서 발라내기 힘들다는 문제도 있지만..

**Reply by BenjaminKim** [reply]
ai가 좋아지면서 점점 쉽게 발라낼 수 있을 것 같아요.
나중에 빛을 발하는 날 기다리며 저도 열심히 기록중입니다. 😁

**Comment by Invidam** [comment]
아이폰에서 찍은 사진들을 영상으로 만들어 년도별/테마별로 보여주는 기능이 있더라구요. 너무 신기했습니다 ㅋㅋ

**Reply by BenjaminKim** [reply]
구글 포토에도 비슷한 기능들이 있어요. 사진 어플들이 점점 좋아지고 있어서 옛날 사진의 가치가 더 올라가는 것 같습니다.

**Comment by roeniss** [comment]
다시 찾으려니까 못찾겠는데... 유튜브에 자동 업로드 되어버린 홈 레코딩 클립들을 모아서 보여주는 웹서비스도 있었던 것 같습니다. 예전에 유튜브에서 그런 기능을 제공했다나요.

**Comment by YONGKALBI** [comment]
안녕하세요 재호님, 저는 한 85년 생으로 사진에 올라온 가리봉 교회, 구로동 - 구로남국민학교 쪽과 관련한 기억을 공유하고 있는 사람입니다. ^^ 
말씀하신 서비스가 절실합니다.. ㅎㅎ 저는 개인적으로 작년부터 사진 - 영상 등을 구하려고 노력하고 있는데, 구하기가 상당히 어렵더라구요, 심지어 구입하고 싶어도 공지할 방법도…

작년부터 추억 프로젝트를 하면서 국민학교 은사님 포함해서 자료를 수집했고, 애플 비전프로로 3D 화해서 열심히 컨텐츠 소비중입니다. 추억이 계속 살아납니다 ㅎㅎ 한번 커피챗 요청드리고 싶네요 ㅎㅎ

**Reply by BenjaminKim** [reply]
안녕하세요, 저랑 거의 동시대에서 같은 공간에서 자라셨군요. 반갑습니다.
마흔 살 쯤 넘어가면 옛 기억이 소중해지게 되는데, 찾을 수가 없다보니 이런 아이디어가 생각나게 되었어요.
비슷한 느낌을 가지신 분이 있다니, 저 혼자만 원하는 건 아니라 다행입니다. 어서 만들어봐야겠다는 생각이 드네요.
커피챗도 좋으니 메일로 연락주세요. (블로그 footer 에 메일 주소가 있습니다.)
